lib/screens/components/hero_card.dart

```diff
@@ -0,0 +1,83 @@
+import 'package:flutter/material.dart';
+
+import '../../domain/models/marvel_hero.dart';
+import '../../themes/app_colors.dart';
+import '../../utils/constants/routes.dart';
+
+class HeroCard extends StatelessWidget {
+  const HeroCard({required this.hero, super.key});
+  final MarvelHero hero;
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.fromLTRB(16, 8, 16, 0),
+      child: Card(
+        color: AppColors.redCard,
+        clipBehavior: Clip.antiAlias,
+        elevation: 10,
+        shadowColor: AppColors.dark,
+        child: Ink(
+          width: 100,
+          child: InkWell(
+            borderRadius: BorderRadius.circular(10),
+            onTap: () =>
+                Navigator.pushNamed(context, Routes.details, arguments: hero),
+            child: Row(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: [
+                Expanded(
+                  child: Image.network(
+                    hero.image,
+                    fit: BoxFit.cover,
+                    height: 120,
+                  ),
+                ),
+                Expanded(
+                  flex: 2,
+                  child: Column(
+                    mainAxisAlignment: MainAxisAlignment.center,
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    children: [
+                      Row(
+                        mainAxisAlignment: MainAxisAlignment.end,
+                        children: [
+                          Padding(
+                            padding: const EdgeInsets.only(right: 8),
+                            child: Text(
+                              hero.id.toString().padLeft(8, '#'),
+                              style: const TextStyle(
+                                color: AppColors.greyId,
+                              ),
+                            ),
+                          ),
+                        ],
+                      ),
+                      Row(
+                        children: [
+                          SizedBox(
+                            width: 230,
+                            height: 80,
+                            child: Center(
+                              child: Text(
+                                hero.name,
+                                textAlign: TextAlign.center,
+                                style: Theme.of(context).textTheme.titleSmall,
+                                overflow: TextOverflow.clip,
+                                softWrap: true,
+                              ),
```

Comment:
Remover a width e usar Expanded. Ele não estava centralizando de verdade.
Evitar usar valores fixos sempre que possível, principalmente pra centralizar
```suggestion
                      Row(
                        children: [
                          Expanded(
                            child: SizedBox(
                              height: 80,
                              child: Center(
```

Reply:
Olha o espaço da direita que não está sendo centralizado.
![Screenshot_1710850560](https://github.com/leooruiz/Marvel_app/assets/61849372/f9a3ff30-080d-4308-8581-cb186e2eacea)